{
  "Header": {
    "Original": "CVE-2020-1695 (A flaw was found in all resteasy 3.x.x versions prior to 3.12.0.Final  ...)",
    "Line": 267327,
    "ID": "CVE-2020-1695",
    "Description": "(A flaw was found in all resteasy 3.x.x versions prior to 3.12.0.Final  ...)"
  },
  "Annotations": [
    {
      "Original": "- resteasy \u003cunfixed\u003e (bug #1034804)",
      "Line": 267328,
      "Type": "package",
      "Package": "resteasy",
      "Kind": "unfixed",
      "BugNo": 1034804
    },
    {
      "Original": "- resteasy3.0 3.0.26-2",
      "Line": 267329,
      "Type": "package",
      "Package": "resteasy3.0",
      "Kind": "fixed",
      "Version": "3.0.26-2"
    },
    {
      "Original": "[buster] - resteasy3.0 \u003cno-dsa\u003e (Minor issue)",
      "Line": 267330,
      "Type": "package",
      "Release": "buster",
      "Package": "resteasy3.0",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1730462",
      "Line": 267331,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1730462"
    },
    {
      "Original": "NOTE: https://github.com/resteasy/Resteasy/commit/acf15f2a8067f7e4cf5838342cecfa0b78a174fb",
      "Line": 267332,
      "Type": "NOTE",
      "Description": "https://github.com/resteasy/Resteasy/commit/acf15f2a8067f7e4cf5838342cecfa0b78a174fb"
    }
  ]
}